kyverno-integration/modules/enforce/exceptions/kind.yaml

```diff
@@ -0,0 +1,66 @@
+apiVersion: kyverno.io/v2beta1
```

Comment:
Do this file name `kind.yaml` make sense? I thought it was referring to kind cluster we use idpbuilder.
Would there be a better name?

Reply:
It is for kind clusters

Reply:
ah specific expections for kind clusters? then is correct